services/src/test/scala/cromwell/pipeline/service/ProjectFileServiceTest.scala

```diff
@@ -45,26 +49,53 @@ class ProjectFileServiceTest extends AsyncWordSpec with Matchers with MockitoSug
     }
 
     "upload file" should {
+      val userId = UserId.random
       val project = TestProjectUtils.getDummyProject()
+      val projectId = project.projectId
       val projectFileContent = ProjectFileContent("File content")
       val projectFile = ProjectFile(Paths.get("test.txt"), projectFileContent)
       val version = TestProjectUtils.getDummyPipeLineVersion()
 
       "return success message for request" taggedAs Service in {
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.successful(project))
         when(projectVersioning.updateFile(project, projectFile, Some(version)))
           .thenReturn(Future.successful(Right(UpdateFiledResponse(projectFile.path.toString, "master"))))
         projectFileService
-          .uploadFile(project, projectFile, Some(version))
+          .uploadFile(projectId, projectFile, Some(version), userId)
           .map(_ shouldBe Right(UpdateFiledResponse(projectFile.path.toString, "master")))
       }
 
       "return error message for error request" taggedAs Service in {
         when(projectVersioning.updateFile(project, projectFile, Some(version)))
           .thenReturn(Future.successful(Left(VersioningException.HttpException("Something wrong"))))
         projectFileService
-          .uploadFile(project, projectFile, Some(version))
+          .uploadFile(projectId, projectFile, Some(version), userId)
           .map(_ shouldBe Left(VersioningException.HttpException("Something wrong")))
       }
     }
+
+    "build configuration" should {
+      val project = TestProjectUtils.getDummyProject()
+      val projectId = project.projectId
+      val projectFileContent = ProjectFileContent("File content")
+      val projectFile = ProjectFile(Paths.get("test.txt"), projectFileContent)
+
+      "return success message for request" taggedAs Service in {
+        when(womTool.inputsToList(projectFileContent.content)).thenReturn(Right(any()))
```

Comment:
Don't use `any()` matcher in as a returned value. These things tend to unexpectedly fail. Just return `Nil` / `List.empty`

Reply:
Ok, tnx. Fixed.